metadeploy/api/flows.py

```diff
@@ -59,6 +59,15 @@ def pre_flow(self, coordinator):
         return self.logger
 
     def post_flow(self, coordinator):
+        """
+        Send a password reset email when completing scratch org JobFlows
+        """
+        org_config = coordinator.org_config
+        if org_config.config.get("scratch"):
```

Comment:
`if org_config.scratch:` is how we would usually write this (OrgConfig and other subclasses of BaseConfig have a `__getattr__` which provides attribute access to the keys stored in `.config`, and returns None if the key is not present)

Reply:
I actually played with getting just the ID from the `org_config` to check against our `ScratchOrg` instances. I just wanted to save a DB query here but it seems like the safer option. Let me know what you think.

Reply:
The `scratch` key in org configs is set by CumulusCI to keep track of which org configs should get loaded as a ScratchOrgConfig that gets credentials by calling sfdx in a subprocess instead of an OrgConfig that has credentials obtained via oauth. So `scratch` is not really expected to be there for any org we're dealing with in MetaDeploy.

I like @jerivas idea.

Reply:
I'll add that in

Reply:
@davisagli I'm now doing `ScratchOrg.objects.get(org_id=config.org_id)` but that fails because `config.org_id` is `00DJ0000003ZYXkMAO` while the id on the ScratchOrg is `00DJ0000003ZYXk` (missing the last 3 letters). Am I checking against the wrong property?

Reply:
@jerivas ids in Salesforce can be in either a 15-character form or an 18-character form (case insensitive). The last 3 characters of the 18-character form are an encoded representation of whether the other 15 characters are upper case or lower case. I'm not quite sure why we're getting different forms in different places here, but it should work okay to truncate the long one to 15 characters and then search for that on the ScratchOrg.

Reply:
@jerivas We also have a `convert_to_18` helper method that could be used here:

- https://github.com/oddbird/MetaDeploy/blob/main/metadeploy/api/belvedere_utils.py#L8-L27
- https://github.com/oddbird/MetaDeploy/blob/main/metadeploy/api/models.py#L110-L113